scilpy/io/utils.py

```diff
@@ -22,6 +22,16 @@ def check_tracts_same_format(parser, filename_list):
             parser.error('All tracts file must use the same format.')
 
 
+def add_json_arg(parser):
```

Comment:
Should probably be `add_json_args` with an `s`, since you have more than 1 arg ;)

Reply:
fix